code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -1817,8 +1817,8 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 
 	return ..()
 
-/mob/living/carbon/human/proc/get_age_pitch()
-	return 1.0 + 0.5*(30 - age)/80
+/mob/living/carbon/human/proc/get_age_pitch(species_pitch = 85)
+	return 1.0 + 0.5 * (30 - age) / (0.94 * species_pitch)
```

Comment:
were these values just what worked? Or is there a reason for these numbers specifically besides that

Reply:
OLD PITCH

https://user-images.githubusercontent.com/77684085/235300475-499b2e5c-d03a-4d6f-b175-3d182f3dbd16.mp4

NEW PITCH

https://user-images.githubusercontent.com/77684085/235300479-88714ae2-84f1-4d3d-9014-9457ee9b3429.mp4

I changed those values assuming that they are percentages based on the maximum age (which originally was 85 years), with 80 being 94% of it and 30 being 35%.